src/reportError.ts

```diff
@@ -69,3 +72,12 @@ function printRelatedSources (msg: Object) {
 function printErrorSummary (message: string) {
   console.log(chalk.red('ERROR'), message)
 }
+
+function reportModifiedDependency (err: Error, msg: Object) {
+  printErrorSummary('Packages in the store have been mutated')
+  console.log()
+  console.log('These packages are modified:')
+  msg['modified'].forEach((pkgPath: string) => console.log(chalk.gray(pkgPath)))
+  console.log()
```

Comment:
do you use empty `console.log`s to get empty line? if so, i'd suggest to go with [`stripIndent` template literal tag from common-tags](https://github.com/declandewet/common-tags#stripindent)